packages/common/mui/src/lib/note-chart/note-chart.tsx

```diff
@@ -137,16 +156,26 @@ export const NoteChart = ({
       padding: 0,
       // TODO: Bring in data format function for axisValue when available
       formatter: function (params) {
-        return `<div style="background-color: #041D2E; width: 100px; box-shadow: -2px 1px 24px 0px rgba(135, 155, 215, 0.20), 0px 4px 16px 0px rgba(121, 209, 212, 0.40); padding: 8px; height: 58px;">
-                  <div style="color: #BCD4DB; font-family: Avenir Next; font-weight: 500;">${params[0].axisValue}</div>
-                  <div style="color: #ffffff; font-family: Avenir Next; font-weight: 500;" >${params[0].data}</div>
+        const value = formatToolTipValueAsFiat
+          ? formatNumberAsAbbr(params[0].data[1], 4, baseCurrency)
+          : formatNumber(params[0].data[1]);
+        if (noteNumCallback) {
+          noteNumCallback(params[0].data[1]);
+        }
+        return `<div style="background-color: #041D2E; width: fit-content; box-shadow: -2px 1px 24px 0px rgba(135, 155, 215, 0.20), 0px 4px 16px 0px rgba(121, 209, 212, 0.40); padding: 8px; height: 58px;">
```

Comment:
Can we make a styled component instead of embedding all this CSS like this?

Reply:
Thought that was something we could do when we extend the chart in the future. We could have a number of different tooltip components like our table cell components to handle different situations. I just didn't take the time to do that right now.

Reply:
After further investigation into the docs I don't think when can use a normal component here. A string template has to be returned. I don't think it handle JSX

Reply:
ok sounds good